docs/application/dotnet/guides/nui/view.md

```diff
@@ -1,75 +1,133 @@
 # View
 
-The `View` class is the base class for all views. With this class you can manage the background color and images for UI components.
+[View](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html) is the fundamental concept for the all components such as `Button`, `Image`, `Text`, and so on.
+`View` provides [Properties](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#properties), [Methods](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#methods), and [Events](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#events) that are commonly used on every components. To show the `View`, you must add `View` on the [Window](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Window.html) as a child.
 
-In this tutorial, the following subjects are covered:
-
-[View events](#1)<br>
-[Setting the Background Color](#2)<br>
-[Setting the Background Image](#3)<br>
 
 <a name="1"></a>
-## View events
+## View Transforms
 
-The following table lists the basic event provided by the `View` class:
+`position`, `orientation`, and `scale` of `View` are known as transforms.
+`View` supports hierarchical structure. Therefore, `position`, `orientation`, and `scale` of every `View` are affected by their parents `Views`.
```

Comment:
Please consider this comment, https://github.com/Samsung/tizen-docs/pull/962#discussion_r353674978.

Reply:
I fixed all.